From: Patrick Brunschwig <patrick@enigmail.net>
Date: Tue, 18 Oct 2016 08:19:13 +0200
Subject: fixed various merge errors

(from upstream 6365abe7df40128a1d1950bf438700bfb6cd3812, except for
the version number change)
---
 Makefile                            | 2 +-
 package/key.jsm                     | 4 +---
 package/mimeDecrypt.jsm             | 4 ++--
 ui/locale/en-US/enigmail.properties | 2 ++
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 11c830e..e314b07 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 XPI_MODULE	= enigmail
-XPI_MODULE_VERS = 1.9.5
+XPI_MODULE_VERS = 1.9.6
 
 DEPTH		= .
 
diff --git a/package/key.jsm b/package/key.jsm
index f90022c..ecdbdae 100644
--- a/package/key.jsm
+++ b/package/key.jsm
@@ -195,9 +195,7 @@ var EnigmailKey = {
    *    - packetStr - String: the packet list as received from GnuPG
    */
   getKeyFileType: function(keyBlockStr) {
-    let args = EnigmailGpg.getStandardArgs(true).concat(["--no-verbose, "
-      "--list-packets"
-    ]);
+    let args = EnigmailGpg.getStandardArgs(true).concat(["--no-verbose, --list-packets"]);
     const exitCodeObj = {};
     const statusMsgObj = {};
     const errorMsgObj = {};
diff --git a/package/mimeDecrypt.jsm b/package/mimeDecrypt.jsm
index 736dee3..5336e8c 100644
--- a/package/mimeDecrypt.jsm
+++ b/package/mimeDecrypt.jsm
@@ -448,7 +448,7 @@ EnigmailMimeDecrypt.prototype = {
         try {
           veri.onDataAvailable(null, null, gConv, 0, data.length + 1);
         }
-        catch (x) {
+        catch (ex) {
           EnigmailLog.ERROR("mimeDecrypt.jsm: returnData(): mimeSvc.onDataAvailable failed:\n" + ex.toString());
         }
         veri.onStopRequest(null, null, 0);
@@ -458,7 +458,7 @@ EnigmailMimeDecrypt.prototype = {
         try {
           this.mimeSvc.onDataAvailable(null, null, gConv, 0, data.length);
         }
-        catch (x) {
+        catch (ex) {
           EnigmailLog.ERROR("mimeDecrypt.jsm: returnData(): mimeSvc.onDataAvailable failed:\n" + ex.toString());
         }
         this.mimeSvc.onStopRequest(null, null, 0);
diff --git a/ui/locale/en-US/enigmail.properties b/ui/locale/en-US/enigmail.properties
index 7f2f3ae..64cafd4 100644
--- a/ui/locale/en-US/enigmail.properties
+++ b/ui/locale/en-US/enigmail.properties
@@ -472,6 +472,8 @@ deleteKeyFailed=The key could not be deleted.
 revokeKeyQuestion=You are about to revoke the key '%S'.\n\nYou will no longer be able to sign with this key, and once distributed, others will no longer be able to encrypt with that key. You can still use the key to decrypt old messages.\n\nDo you want to proceed?
 revokeKeyOk=The key has been revoked. If your key is available on a key server, it is recommended to re-upload it, so that others can see the revocation.
 revokeKeyFailed=The key could not be revoked.
+revokeKeyAlreadyRevoked=This key has already been revoked.
+revokeKeyNotPresent=No suitable key available for revocation certificate.
 refreshAllQuestion=You did not select any key. Would you like to refresh ALL keys?
 refreshKey.warn=Warning: depending on the number of keys and the connection speed, refreshing all keys could be quite a lengthy process!
 downloadContactsKeys.warn=Warning: depending on the number of contacts and the connection speed, downloading all keys could be quite a lengthy process!
